packages/datatrak-web/src/views/ReportsPage.tsx

```diff
@@ -37,7 +34,47 @@ const Text = styled(Typography)`
   line-height: 1.56;
 `;
 
+const MobileContainer = styled(Paper).attrs({
+  elevation: 0,
+})`
+  text-align: center;
+  max-width: 19rem;
+  padding: 0.5rem 0.5rem 0;
+
+  h1.MuiTypography-root {
+    margin-block-end: 1rem;
+  }
+  p.MuiTypography-root {
+    margin-block-end: 1.5rem;
+  }
+  a.MuiButtonBase-root {
+    width: 100%;
+  }
+`;
+
+const MobileTemplate = () => {
+  const navigate = useNavigate();
+  const onClose = () => navigate('/');
+  return (
+    <Modal open={true} onClose={onClose}>
+      <MobileContainer>
+        <Typography variant="h1">Reports not available on mobile</Typography>
+        <Typography>
+          The reports feature is only available on desktop. Please visit Tupaia Datatrak on desktop
+          to proceed.
+        </Typography>
+        <Button to="/">Close</Button>
```

Comment:
Hm, if this button actually navigates the user back to DataTrak Home, could this label be a bit misleading?

```suggestion
        <Button to="/">Close</Button>
```

Perhaps **Go home** or **Back home** or similar?